frontend/src/templates/Field/ChildrenCompound/ChildrenCompoundField.tsx

```diff
@@ -94,9 +93,9 @@ export const ChildrenCompoundField = ({
   // Initialize with a single child section
   useEffect(() => {
     if (!fields || !fields.length) {
-      append([''])
+      update(0, '')
```

Comment:
Saw that there's another parameter that we can set on append, would this work?

```suggestion
      append([''], { shouldFocus: false })
```

From the docs, update would do a re-render as it unmounts and remounts the component (`<ChildrenBody />`) which might be expensive.

Reply:
ooh yeah, this works. thanks ken! where did you find the other parameters btw? I see `focusOptions` in the docs, but not what they are

Reply:
Similarly, I tried to look at the docs, wasn't helpful. Then I looked into the typedocs for `append` and found it after running through a few rabbitholes 😆 .